Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityDeviceManager.cs

```diff
@@ -754,9 +836,90 @@ public void RemoveControllerForSuspendWorkaround(InteractionSource interactionSo
             RemoveController(interactionSource);
         }
 
-        #endregion Controller Utilities
 
-        #region Unity InteractionManager Events
+#if HP_CONTROLLER_ENABLED
+        private void AddController(object sender, MotionController e)
```

Comment:
Slight nit: `e` isn't a super descriptive parameter name, especially for something of type `MotionController`!